isis/src/base/objs/ProcessExportPds4/ProcessExportPds4.cpp

```diff
@@ -882,36 +898,69 @@ namespace Isis {
         elementArrayElement.appendChild(offsetElement);
       }
 
-      // Add the Special_Constants class to define ISIS special pixel values: 
+      // Add the Special_Constants class to define ISIS special pixel values if pixel type is
+      // Real 
+      if (p_pixelType == Isis::Real) {
```

Comment:
We know we are going to have to support PDS4 for a lot of different missions. This can be easily expanded to work with any ISIS pixel type by using a switch statement.